_i18n/nb-no/resources/user-guides/multisig-messaging-system.md

```diff
@@ -0,0 +1,864 @@
+    --bitmessage-login		Specify <arg> as username:password for PyBitmessage API
+
+Hvis du kjører PyBitmessage på samme maskin som CLI-lommeboken, holder standardverdien for den første parameteren, og du bør ikke trenge å endre på innstillingene. Hvis det ikke virker som den finner den, til tross for å ha kjørt lokalt, kan du prøve å bruke `http://localhost` eller `http://127.0.0.1` som argument for den første parameteren.
+
+Bortsett fra det, trenger du selvfølgelig også enten `--testnet` eller `--stagenet` for å koble til det riktige nettverket. Det kan også være nyttig å bruke `--log-level 0`: Dette instruerer lommeboken til å skrive detaljert info til loggfilen som kan hjelpe med å finne programfeil eller andre problemer med MMS-en.
+
+En komplett kommandolinje for CLI-lommeboken kan altså se slik ut:
+
+    ./monero-wallet-cli --testnet --bitmessage-login mmstest:p4ssw0rd --log-level 0
+
+### Initialisering av MMS-en
+
+Etter å ha opprettet en ny lommebok, må du initialisere den for bruk med MMS-en; uten det essensielle førstetrinnet vil du ikke kunne bruke MMS-funksjonene. Kommandoen for å gjøre dette er `mms init`:
+
+    mms init <required_signers>/<authorized_signers> <own_label> <own_transport_address>
+
+`own_transport_address` er Bitmessage-adressen som du konfigurerte i ditt eget PyBitmessage-program. En full `init`-kommando kan se slik ut:
+
+    mms init 2/2 alice BM-2cUVEbbb3H6ojddYQziK3RafJ5GPcFQv7e
+
+Bruk kun denne `init`-kommanden **én gang**: Å eksekvere den én gang til reinitialiserer MMS-en fullstendig ved å slette eventuell signererinformasjon og meldinger, noe du ikke ønsker, bortsett fra i spesielle tilfeller.
+
+Hvis du vil gjennomgå en MMS-test så raskt som mulig, kan du instruere lommeboken om å be om passordet kun når det er strengt nødvendig av tekniske årsaker og be MMS-en om å sende eventuelle genererte meldinger umiddelbart i stedet for å spørre om det i forkant:
+
+    set ask-password 0
+    mms set auto-send 1
+
+(Begge disse innstillingene er aktive under 2/2 multisig-eksempelet som er vist i denne veiledningen.)
+
+### Konfigurering av signerere
+
+For hver signerer, er det tre ting MMS-en trenger å vite:
+
+* Ordet *merkelapp* som du bruker for å henvise til signereren
+* *Transport-adressen*, som for øyeblikket betyr deres Bitmessage-adresse så lenge dette er det eneste støttede meldingstransportsystemet
+* *Monero-adressen*, altså den «opprinnelige» Monero-adressen til lommeboken deres
+
+(Se også kapittelet over som heter *Adresser og merkelapper*.)
+
+Du trenger ikke å opprette signerere; de er allerede «der» etter `mms init`-kommandoen, men det ikke finnes informasjon om dem ennå, bortsett fra om deg. Kommandoene for å sette signererinformasjonen går etter nummer: 1 opptil det totalte antallet autoriserte signerere, altså 1 og 2 i følgende 2/2 multisig-eksempelet der signererne heter *Alice* og *Bob*, og altså med merkelappene *alice* og *bob*.

```

Comment:
Fix typo: totale antallet* + add "det" before "følgende"

Reply:
Fixed. I changed to «… i følgende 2/2 multisig-eksempel»